timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -1046,3 +1046,43 @@ def refit_full(self, model: str = "all", set_best_to_refit_full: bool = True) ->
                     f"Training may have failed on the refit model. AutoGluon will default to using '{model_best}' for predict()."
                 )
         return refit_full_dict
+
+    def get_simulation_artifact(self, test_data: TimeSeriesDataFrame) -> dict:
```

Comment:
I renamed to `simulation_artifact` for the Tabular code, may want to do the same here.